.golangci.yaml

```diff
@@ -80,7 +80,6 @@ issues:
     - hack
     - charts
     - designs
-    - pkg/alt/knative # copy
     - pkg/alt/karpenter-core/pkg/webhooks # copy
   exclude-files:
     - pkg/alt/karpenter-core/pkg/operator/logger.go # copy
```

Comment:
Can we remove the other two pkg/alt as well? I missed this

Reply:
yeah. I actually already removed these in another branch but I'll remove it here too (cleans up the future PR too anyway).